src/components/Home.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function Home() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.

Reply:
Thank you for the good suggestion. Will change them to arrow functions.